### DISCUSSION TITLE
Viber

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#62] [General]
# Viber

Check if Viber is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/eabeeda0-6cf8-45f2-b8ee-3648017b3d6e

**Comment by ppavle** [comment]
Viber is not native app.  On their site they do say that download is for ARM too, but I guess they meant that is works in Prism (emulaton).
I downloaded it, installed it. It's x64.
"Download Viber for 64-bit version of Windows 10, Windows 11 and Windows 11 on ARM"

**Reply by OpenSource03** [reply]
Correct, updated :)

P.S. Pozdrav iz Srbije, nisam ocekivao da neko sa nasih prostora koristi WOA 😅

**Reply by ppavle** [reply]
Surface Laptop 7 15 inch :). A još 2013 imao Surface RT :)
